### PR TITLE
fix (sipi): bump sipi and add missing config (memory leak) (DEV-237)

### PR DESCRIPTION
**Pull request by subotic**
resolves DEV-237

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -26,13 +26,13 @@ services:
...
     networks:
       - knora-net
-    restart: unless-stopped
     environment:
       - SIPI_EXTERNAL_PROTOCOL=http
       - SIPI_EXTERNAL_HOSTNAME=localhost
       - SIPI_EXTERNAL_PORT=1024
       - SIPI_WEBAPI_HOSTNAME=api
       - SIPI_WEBAPI_PORT=3333
+    # entrypoint: [ "valgrind", "--leak-check=yes", "/sipi/sipi" ] ## uncomment to run SIPI under valgrind
```
Comment (irinaschubert) [inline docker-compose.yml:35]: What is valgrind? In what cases do we use it?

File: sipi/config/sipi.knora-docker-config.lua [hunk 2]
```diff
@@ -22,6 +22,34 @@ sipi = {
...
+    --
+    -- SIPI is using libjpeg to generate the JPEG images. libjpeg requires a quality value which
+    -- corresponds to the compression rate. 100 is (almost) no compression and best quality, 0
+    -- would be full compression and no quality. Reasonable values are between 30 and 95...
+    --
+    jpeg_quality = 60,
+
+    --
+    -- For scaling images, SIPI offers two methods. The value "high" offers best quality using expensive
+    -- algorithms (bilinear interpolation, if downscaling the image is first scaled up to an integer
```
Comment (irinaschubert) [inline sipi/config/sipi.knora-docker-config.lua:39]:
```suggestion
    -- algorithms: bilinear interpolation, if downscaling the image is first scaled up to an integer
```

File: sipi/config/sipi.knora-docker-config.lua [hunk 3]
```diff
@@ -22,6 +22,34 @@ sipi = {
...
+    -- multiple of the requires size, and then downscaled using averaging. This results in the best
+    -- image quality. "medium" uses bilinear interpolation but does not do upscaling before
+    -- downscaling. Scaling quality is set to "low", then just a lookup table and nearest integer
```
Comment (irinaschubert) [inline sipi/config/sipi.knora-docker-config.lua:42]:
```suggestion
    -- downscaling. If scaling quality is set to "low", then just a lookup table and nearest integer
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,4 +1,47 @@
-===REMOVE===
-Important! Please follow the new guidelines for naming Pull Requests: https://docs.dasch.swiss/developers/dsp/contribution/#pull-request-guidelines
-===REMOVE===
-resolves DEV-
+<!-- Important! Please follow the new guidelines for naming Pull Requests: https://docs.dasch.swiss/developers/dsp/contribution/#pull-request-guidelines-->
```
Comment (irinaschubert) [inline .github/pull_request_template.md:1]: Do we use this template in the future or was it introduced by accident?

**Review by irinaschubert — APPROVED** [review]
LGTM

File: docker-compose.yml (hunk 1 above)
Comment (subotic) [reply docker-compose.yml:35]: valgrind is for finding memory leaks in C++. This is for debugging Sipi while running inside our stack.

File: .github/pull_request_template.md (hunk 4 above)
Comment (subotic) [reply .github/pull_request_template.md:1]: I added it a few weeks ago to try it out. If we don't like it we can remove it. I find it at times difficult to review things without knowing the context.

**Comment by subotic** [comment]
@irinaschubert Thanks for the review :-)

**Pull request merged**